drivers/xaudio2/audio_driver_xaudio2.cpp

```diff
@@ -45,12 +45,12 @@ Error AudioDriverXAudio2::init() {
 	pcm_open = false;
 	samples_in = NULL;
 
-	mix_rate = 48000;
+	mix_rate = GLOBAL_DEF_RST("audio/mix_rate", DEFAULT_MIX_RATE);
```

Comment:
If this was hardcoded to 48000 on purpose (e.g. it's a known requirement for UWP), I can change it back, but we should put a comment.

Reply:
I think this change should be tested just in case.

Reply:
Now that I have a proper Windows 10 system, I might toy around with UWP to see how's the workflow and what is the current state of the port. But I think I'll merge now and test later, if nobody beats me to it (cc @vnen).